backend/src/backend/app/logger.py

```
@@ -0,0 +1,54 @@
+    """
+    Log an exception's full traceback with context information
+    
+    Args:
+        e: The exception to log
+        context: Additional context about where/why the error occurred
+    """
+    # Get the full traceback
+    tb_lines = traceback.format_exception(type(e), e, e.__traceback__)
+    
+    # Format the error message
+    error_msg = f"Exception occurred"
```

Ruff (0.8.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)